modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc

```diff
@@ -36,15 +36,15 @@ metadata:
   annotations:
     che.eclipse.org/oauth-scm-server: github
     che.eclipse.org/scm-server-endpoint: __<github_server_url>__ <2>
-    che.eclipse.org/scm-github-disable-subdomain-isolation: true <3>
+    che.eclipse.org/scm-github-disable-subdomain-isolation: 'false' <3>
 type: Opaque
 stringData:
   id: __<GitHub_OAuth_Client_ID>__ <4>
   secret: __<GitHub_OAuth_Client_Secret>__ <5>
 ----
 <1> The {prod-short} namespace. The default is `{prod-namespace}`.
 <2> This depends on the GitHub product your organization is using: When hosting repositories on GitHub.com or GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`. When hosting repositories on GitHub Enterprise Server, enter the GitHub Enterprise Server URL.
-<3> This line is only added for GitHub Enterprise Server with disabled link:https://docs.github.com/en/enterprise-server/admin/configuration/hardening-security-for-your-enterprise/enabling-subdomain-isolation#about-subdomain-isolation[subdomain isolation] option. If the subdomain isolation option is enabled on GitHub Enterprise Server, you must either omit this annotation or set it to `false`.
+<3> If you are using GitHub Enterprise Server with disabled link:https://docs.github.com/en/enterprise-server/admin/configuration/hardening-security-for-your-enterprise/enabling-subdomain-isolation#about-subdomain-isolation[subdomain isolation] option, you must set the annotation to `true`, otherwise you can either omit the annotation, or set it to 'true'.
```

Comment:
```suggestion
<3> If you are using GitHub Enterprise Server with a disabled link:https://docs.github.com/en/enterprise-server/admin/configuration/hardening-security-for-your-enterprise/enabling-subdomain-isolation#about-subdomain-isolation[subdomain isolation] option, you must set the annotation to `true`, otherwise you can either omit the annotation or set it to 'false'.
```